README.md

```diff
@@ -1648,3 +1648,101 @@ PS > (echo $warnings | ConvertTo-Json) | seqcli signal update --json-stdin
+
+```shell
+seqcli forwarder run --pre --listen http://127.0.0.1:5341 -s https://seq.example.com
+```
+
+> While the `forwarder` command group is in preview, all `forwarder` commands require the `--pre` switch; you'll
+> also need to supply `--pre` when requesting help, e.g. `seqcli help forwarder run --pre`.
+
+You can test your forwarder using the `seqcli log` command:
+
+```shell
+seqcli log -m Test -s http://127.0.0.1:5341
+```
+
+If forwarding is successful, the event will appear in the target Seq instance within a few seconds. If it isn't,
+and `seqcli log` didn't report an error, run through these troubleshooting steps:
+
+ 1. Read the output of the `seqcli forwarder run` command: are the paths and URLs in there the ones you expect?
+ 2. Check the destination Seq server's ingestion log; if the payload is being rejected by Seq, its likely you need to configure an API key for the forwarder's outbound connections, or API key forwarding (see below).
+ 3. Check the forwarder's log file; this will be in the `SeqCli/Logs` subdirectory of the forwarder's storage path, which will be your user's home directory unless you overrode it with `--storage` or `SEQCLI_STORAGE_PATH`.
+ 4. Enable the forwarder's ingestion log temporarily, ensuring this isn't accessible to untrusted callers.
+
+### The forwarder's ingestion log
+
+The ingestion log records ingestion issues in a short in-memory buffer, so repetitive ingestion issues don't chew up
+disk space. The ingestion log may contain fragments of event data so it's not enabled by default; you can turn it on
+with:
+
+```shell
+SEQCLI_FORWARDER_DIAGNOSTICS_EXPOSEINGESTIONLOG=True seqcli forwarder run <other args>
+```
+
+The log can be retrieved by pointing `seqcli` at the forwarder:
+
+```shell
+seqcli diagnostics ingestionlog -s http://127.0.0.1:5341
+```
+
+If the ingestion log contains relevant entries but doesn't include enough information to track down the issue, you
+can opt in to showing even more information with `SEQCLI_FORWARDER_DIAGNOSTICS_INGESTIONLOGSHOWDETAIL=True`.
+
+### Configuring the forwarder
+
+The forwarder reads its configuration from the environment and the `SeqCli.json` file. Run `seqcli config` to view
+all of the supported `forwarder.*` (`SEQCLI_FORWARDER_*`) settings.
+
+Locally buffered events are stored in the `SeqCli/Buffer` subdirectory of the directory containing `SeqCli.json`. By 
+default, this will be your user account's home folder.
+
+To change the location of the `SeqCli.json` config file and local buffers, specify the `--storage` argument
+or set the `SEQCLI_STORAGE_PATH` environment variable when configuring and running the forwarder.
+
+### Connection authentication and API key forwarding
+
+When connecting to Seq, `seqcli forwarder` will ignore any API keys attached to incoming payloads, and instead use the server URL 
+and API key stored in `SeqCli.json`, in the `SEQCLI_CONNECTION_*` environment variables, or passed on 
```

Comment:
Should profiles be mentioned here?

Reply:
Hmm, interesting; we should probably add explicit `--profile` support to the `forwarder install` command...

Currently there's nothing forwarder-specific to include about profiles here that I can think of; I'll loop back and close the gap mentioned above, let me know if there's something else I should add to the README 👍